packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -584,11 +592,13 @@ export = (app: Application) => {
     const owner = context.payload.repository.owner.login;
     const repo = context.payload.repository.name;
     const prNumber = context.payload.issue.number;
-    const prResponse = await context.github.pulls.get({
+    const prResponse = await context.octokit.pulls.get({
       owner: owner,
       repo: repo,
       pull_number: prNumber,
     });
+    // This line is giving us errors because the payload is
+    // a comment, I'm not sure how to fix this
     context.payload.pull_request = prResponse.data;
```

Comment:
@tmatsuo, this line is erroring because the payload should be a comment. I'm not sure what you're trying to do here, how can we fix this?

Reply:
It is to pass the pull request data into `scanPullRequest` function.

I presume octokit now has stricter types.

Maybe you can refactor `scanPullRequest`, to accept `prResponse.data` as well?